claat/parser/trim.go

```diff
@@ -174,6 +174,21 @@ func splitSpaceRight(s string) (v string, sp string) {
 	return string(rs), ""
 }
 
+func requiresSpacer(a, b types.Node) bool {
+	t1, ok1 := a.(*types.TextNode)
+	t2, ok2 := b.(*types.TextNode)
+
+	if !(ok1 && ok2) {
+		return false
+	}
+
+	if (t1.Bold && t1.Bold == t2.Bold) || (t1.Italic && t1.Italic == t2.Italic) {
```

Comment:
Return this expression instead of using a conditional

Reply:
Ugh duh - fixed.